daphne/src/messages.rs

```diff
@@ -16,6 +16,8 @@ const KEM_ID_X25519_HKDF_SHA256: u16 = 0x0020;
 const KEM_ID_P256_HKDF_SHA256: u16 = 0x0010;
 const KDF_ID_HKDF_SHA256: u16 = 0x0001;
 const AEAD_ID_AES128GCM: u16 = 0x0001;
+const QUERY_TYPE_TIME_INTERVAL: u8 = 0x01;
+const QUERY_TYPE_FIXED_SIZE: u8 = 0x02;
```

Comment:
I think these both should be `u16`s:

```
enum {
   reserved(0), /* Reserved for testing purposes */
   time_interval(1),
   fixed_size(2),
   (65535)
} QueryType;
```

Reply:
Oh god, good catch. Why did we make this 2 bytes instead of just one?